data/values.yml

```diff
@@ -73935,3 +73935,55 @@
   name: Roll-out tray
   friendly_id: furniture/fixture_features__roll_out_tray
   handle: furniture-fixture-features__roll-out-tray
+- id: 26136
+  name: Caffeine content
+  friendly_id: caffeine_content__regular
+  handle: caffeine-content__regular
+- id: 26137
+  name: Caffeine content
+  friendly_id: caffeine_content__decaf
+  handle: caffeine-content__decaf
+- id: 26300
+  name: Caffeine content
+  friendly_id: caffeine_content__low_caffeine
+  handle: caffeine-content__low_caffeine
+- id: 26138
+  name: Grind size
+  friendly_id: grind_size__extra_coarse
+  handle: grind-size__extra-coarse
+- id: 26139
+  name: Grind size
+  friendly_id: grind_size__coarse
+  handle: grind-size__coarse
```

Comment:
Name field should include the value names, e.g. Extra coarse, Coarse, etc. (instead of the attribute name).
We need to fix all these.